src/osd/PG.cc

```diff
@@ -4361,9 +4361,7 @@ void PG::scrub_unreserve_replicas()
   }
 }
 
-void PG::_scan_rollback_obs(
```

Comment:
i'd suggest update the range of scrub chunk size to be more specific, instead of [5, 25], it should be [osd_scrub_chunk_min, osd_scrub_chunk_max]

Reply:
@tchaikov Repushed, thanks!